src/libs/kata-types/src/annotations/dockershim.rs

```diff
@@ -6,8 +6,18 @@
 
 #![allow(missing_docs)]
 
+//! Copied from k8s.io/pkg/kubelet/dockershim/docker_service.go, used to identify whether a docker
+//! container is a sandbox or a regular container, will be removed after defining those as public
+//! fields in dockershim.
+
+///  ContainerTypeLabelKey is the container type (podsandbox or container) annotation.
```

Comment:
Label or annotation? should the name to be adjusted to the type of the key?

Reply:
fixed in next version